packages/mongo-nav/src/data.ts

```diff
@@ -26,7 +26,7 @@ export const dataFixtures = {
     orgId: 'fakeOrgId1',
     planType: 'atlas',
     projectId: 'fakeProjectId1',
-    projectName: 'Test Project',
+    projectName: 'Demo Project 1',
```

Comment:
[background] This data fixture had to be updated because it didn't match any project in the `currentProjects` data fixtures.